### ISSUE TITLE
Rails unable to load with gem

### ISSUE THREAD

**Issue by zacksiri** [librato/librato-rails#106]
Hey Guys,

I am running into issues with the gem 

There was an error while trying to load the gem 'librato-rails'

```
/usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/bundler-1.11.2/lib/bundler/runtime.rb:80:in `rescue in block (2 levels) in require': There was an error while trying to load the gem 'librato-rails'. (Bundler::GemRequireError)
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/bundler-1.11.2/lib/bundler/runtime.rb:76:in `block (2 levels) in require'
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/bundler-1.11.2/lib/bundler/runtime.rb:72:in `each'
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/bundler-1.11.2/lib/bundler/runtime.rb:72:in `block in require'
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/bundler-1.11.2/lib/bundler/runtime.rb:61:in `each'
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/bundler-1.11.2/lib/bundler/runtime.rb:61:in `require'
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/bundler-1.11.2/lib/bundler.rb:99:in `require'
	from /Users/zacksiri/Repositories/lambda/config/application.rb:6:in `<top (required)>'
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/spring-1.6.4/lib/spring/application.rb:82:in `require'
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/spring-1.6.4/lib/spring/application.rb:82:in `preload'
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/spring-1.6.4/lib/spring/application.rb:143:in `serve'
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/spring-1.6.4/lib/spring/application.rb:131:in `block in run'
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/spring-1.6.4/lib/spring/application.rb:125:in `loop'
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/spring-1.6.4/lib/spring/application.rb:125:in `run'
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/spring-1.6.4/lib/spring/application/boot.rb:18:in `<top (required)>'
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
```

**Comment by nextmat** [comment]
@zacksiri - thanks for the report, can you tell us exactly what ruby and rails versions you are using?

**Comment by josephruscio** [comment]
@chancefeick can you take a look at this once we get the details?

**Comment by chancefeick** [comment]
@josephruscio ack

**Comment by zacksiri** [comment]
@nextmat rails 4.2.5.1, ruby 2.3.0

**Comment by chancefeick** [comment]
@zacksiri Which version of `librato-rails` do you have installed? Ruby 2.3.0 isn’t officially supported in our build matrix yet, however I used a test branch for our CI against 2.3.0 and it would not reproduce the error: https://travis-ci.org/librato/librato-rails/jobs/114390314

Do you mind sharing the success output from `bundle install`?

**Comment by zacksiri** [comment]
seems to be using 1.1.0

```
Fetching gem metadata from https://rubygems.org/..........
Fetching version metadata from https://rubygems.org/...
Fetching dependency metadata from https://rubygems.org/..
Resolving dependencies....
Using rake 10.5.0
Using i18n 0.7.0
Using json 1.8.3
Using minitest 5.8.4
Using thread_safe 0.3.5
Using builder 3.2.2
Using erubis 2.7.0
Using mini_portile2 2.0.0
Using rack 1.6.4
Using mime-types 2.99.1
Using arel 6.0.3
Using addressable 2.4.0
Using aggregate 0.2.2
Using bundler 1.11.2
Using thor 0.19.1
Using concurrent-ruby 1.0.1
Using ansi 1.5.0
Using execjs 2.6.0
Using jmespath 1.1.3
Using ice_nine 0.11.2
Using bcrypt 3.1.11
Using debug_inspector 0.0.2
Using sass 3.4.21
Using byebug 8.2.2
Using chunky_png 1.3.5
Using rubyzip 1.2.0
Using coderay 1.1.1
Using coffee-script-source 1.10.0
Using multi_json 1.11.2
Using rb-fsevent 0.9.7
Using ffi 1.9.10
Using tilt 2.0.2
Using connection_pool 2.2.0
Using unf_ext 0.0.7.2
Using multipart-post 2.0.0
Using elasticsearch-rails 0.1.8
Using equalizer 0.0.11
Using geocoder 1.3.1
Using hashie 3.4.3
Using jwt 1.5.1
Using multi_xml 0.5.5
Using request_store 1.3.0
Using google-analytics-turbolinks 0.0.4
Using hetchy 1.0.0
Using kubernetes-secret-rails 0.2.2
Using method_source 0.8.2
Using mini_magick 4.4.0
Using ruby-progressbar 1.7.5
Using sixarm_ruby_unaccent 1.1.1
Using netrc 0.11.0
Using noty-rails 2.3.8
Using nprogress-rails 0.1.6.7
Using oauth 0.5.1
Using pg 0.18.4
Using slop 3.6.0
Using puma 3.1.0
Using redis 3.2.2
Using selectize-rails 0.12.1
Using spring 1.6.4
Using statesman 2.0.0
Using underscore-rails 1.8.3
Using yajl-ruby 1.2.1
Using postmark 1.7.1
Using rdoc 4.2.2
Using tzinfo 1.2.2
Using descendants_tracker 0.0.4
Using sitemap_generator 5.1.0
Using nokogiri 1.6.7.2
Using rack-test 0.6.3
Using font_assets 0.1.12
Using rack-accept 0.4.5
Using rack-mount 0.8.3
Using rack-protection 1.5.3
Using mail 2.6.3
Using sprockets 3.5.2
Using autoprefixer-rails 6.3.3.1
Using uglifier 2.7.2
Using aws-sdk-core 2.2.24
Using binding_of_caller 0.7.2
Using compass-import-once 1.0.5
Using city-state 0.0.13
Using coffee-script 2.4.1
Using compass-core 1.0.3
Using elasticsearch-api 1.0.15
Using rollbar 2.7.1
Using rb-inotify 0.9.7
Using unf 0.1.4
Using faraday 0.9.2
Using minitest-reporters 1.1.8
Using money 6.7.0
Using pry 0.10.3
Using redis-store 1.1.7
Using redis-namespace 1.5.2
Using sidekiq 4.1.1
Using sdoc 0.4.1
Using activesupport 4.2.5.1
Using axiom-types 0.1.1
Using coercible 1.0.0
Using loofah 2.0.3
Using sinatra 1.4.7
Using bootstrap-sass 3.3.6
Using aws-sdk-resources 2.2.24
Using compass 1.0.3
Using domain_name 0.5.20160216
Using elasticsearch-transport 1.0.15
Using faraday-http-cache 1.2.2
Using oauth2 1.1.0
Using librato-metrics 1.6.0
Using monetize 1.4.0
Using pry-rails 0.3.4
Using redis-rack 1.5.0
Using rails-deprecated_sanitizer 1.0.3
Using globalid 0.3.6
Using activemodel 4.2.5.1
Installing clockwork 1.2.0
Using jbuilder 2.4.1
Using redis-activesupport 4.1.5
Using virtus 1.0.5
Using rails-html-sanitizer 1.0.3
Using aws-sdk 2.2.24
Using http-cookie 1.0.2
Using elasticsearch 1.0.15
Using github_api 0.13.1 from git://github.com/zacksiri/github.git (at master@0c9ae37)
Using sorcery 0.9.1
Using librato-rack 0.6.0
Using rails-dom-testing 1.0.7
Using activejob 4.2.5.1
Using activerecord 4.2.5.1
Using rails-observers 0.1.2
Using tilt-jbuilder 0.7.1
Using grape 0.14.0
Using rest-client 1.8.0
Using searchkick 1.2.1
Using actionview 4.2.5.1
Using after_commit_action 1.0.1
Using ancestry 2.1.0
Using annotate 2.7.0
Using friendly_id 5.1.0
Using audited 4.2.0
Using grape-jbuilder 0.2.0
Using refile 0.6.2
Using actionpack 4.2.5.1
Using counter_culture 0.1.34
Using audited-activerecord 4.2.0
Using refile-memory 0.2.0
Using refile-mini_magick 0.2.0
Using refile-s3 0.2.0
Using actionmailer 4.2.5.1
Using railties 4.2.5.1
Using sprockets-rails 3.0.4
Using kaminari 0.16.3
Using gon 6.0.1
Using redis-actionpack 4.0.1
Using postmark-rails 0.12.0
Using coffee-rails 4.1.1
Using font-awesome-rails 4.5.0.1
Using jquery-rails 4.1.0
Using librato-rails 1.1.0
Using money-rails 1.6.0
Using quiet_assets 1.1.0
Using rails 4.2.5.1
Using sass-rails 5.0.4
Using web-console 2.3.0
Using redis-rails 4.0.0
Using transponder 2.0.0
Using turbolinks 2.5.3
Using animate-rails 1.0.9
Using bootstrap-kaminari-views 0.0.5
Using dropzonejs-rails 0.7.3
Using gretel 3.0.8
Using thor-rails 0.0.1
Using trix 0.9.5
```

**Comment by zacksiri** [comment]
@chancefeick whats strange is if i do 

```ruby
gem 'librato-rails', require: false
```

and then run my console and 

```ruby
require 'librato-rails'
```

separately it loads it just fine.

**Comment by zacksiri** [comment]
more details, 

i manually require 'librato-rails' in the application.rb here is something useful

```ruby
(erb):2:in `<main>': undefined method `secrets' for nil:NilClass (NoMethodError)
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/2.3.0/erb.rb:864:in `eval'
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/2.3.0/erb.rb:864:in `result'
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/librato-rails-1.1.0/lib/librato/rails/configuration.rb:43:in `configure_with_config_file'
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/librato-rails-1.1.0/lib/librato/rails/configuration.rb:27:in `load_configuration'
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/librato-rack-0.6.0/lib/librato/rack/configuration.rb:30:in `initialize'
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/librato-rails-1.1.0/lib/librato/rails/configuration.rb:20:in `initialize'
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/librato-rails-1.1.0/lib/librato/rails/railtie.rb:9:in `new'
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/librato-rails-1.1.0/lib/librato/rails/railtie.rb:9:in `<class:Railtie>'
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/librato-rails-1.1.0/lib/librato/rails/railtie.rb:3:in `<module:Rails>'
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/librato-rails-1.1.0/lib/librato/rails/railtie.rb:2:in `<module:Librato>'
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/librato-rails-1.1.0/lib/librato/rails/railtie.rb:1:in `<top (required)>'
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/librato-rails-1.1.0/lib/librato/rails.rb:9:in `require_relative'
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/librato-rails-1.1.0/lib/librato/rails.rb:9:in `<top (required)>'
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/activesupport-4.2.5.1/lib/active_support/dependencies.rb:274:in `require'
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/activesupport-4.2.5.1/lib/active_support/dependencies.rb:274:in `block in require'
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/activesupport-4.2.5.1/lib/active_support/dependencies.rb:240:in `load_dependency'
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/activesupport-4.2.5.1/lib/active_support/dependencies.rb:274:in `require'
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/librato-rails-1.1.0/lib/librato-rails.rb:1:in `<top (required)>'
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/activesupport-4.2.5.1/lib/active_support/dependencies.rb:274:in `require'
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/activesupport-4.2.5.1/lib/active_support/dependencies.rb:274:in `block in require'
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/activesupport-4.2.5.1/lib/active_support/dependencies.rb:240:in `load_dependency'
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/activesupport-4.2.5.1/lib/active_support/dependencies.rb:274:in `require'
	from /Users/zacksiri/Repositories/lambda/config/application.rb:8:in `<top (required)>'
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/spring-1.6.4/lib/spring/application.rb:82:in `require'
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/spring-1.6.4/lib/spring/application.rb:82:in `preload'
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/spring-1.6.4/lib/spring/application.rb:143:in `serve'
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/spring-1.6.4/lib/spring/application.rb:131:in `block in run'
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/spring-1.6.4/lib/spring/application.rb:125:in `loop'
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/spring-1.6.4/lib/spring/application.rb:125:in `run'
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/spring-1.6.4/lib/spring/application/boot.rb:18:in `<top (required)>'
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /usr/local/var/rbenv/versions/2.3.0/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from -e:1:in `<main>'
```

**Comment by zacksiri** [comment]
@chancefeick pull request

https://github.com/librato/librato-rails/pull/107

**Comment by nextmat** [comment]
Closed via #107.
